src/network/networkd-util.h

```diff
@@ -6,10 +6,10 @@
 
 typedef enum AddressFamilyBoolean {
         /* This is a bitmask, though it usually doesn't feel that way! */
-        ADDRESS_FAMILY_NO = 0,
-        ADDRESS_FAMILY_IPV4 = 1,
-        ADDRESS_FAMILY_IPV6 = 2,
-        ADDRESS_FAMILY_YES = 3,
+        ADDRESS_FAMILY_NO,
+        ADDRESS_FAMILY_IPV4,
+        ADDRESS_FAMILY_IPV6,
+        ADDRESS_FAMILY_YES,
```

Comment:
hmm, i think it would make sense to assign the values explicit here like this:

```
        ADDRESS_FAMILY_NO   = 0,
        ADDRESS_FAMILY_IPV4 = 1 << 0,
        ADDRESS_FAMILY_IPV6 = 1 << 1,
        ADDRESS_FAMILY_YES  = ADDRESS_FAMILY_IPV4|ADDRESS_FMAILY_IPV6,
```

Which is of course actually the same, but it underlines that this is actually an enum that also happens to be a bitmask.

Reply:
Sounds nice!